common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -360,9 +360,24 @@ public static void info(final String tag, @Nullable final String message) {
     public static void info(final String tag,
                             @Nullable final String correlationID,
                             @Nullable final String message) {
-        getInstance().log(tag, LogLevel.INFO, correlationID, message, null, false);
+        info(tag, correlationID, message, null);
     }
 
+    /**
+     * * Send a {@link LogLevel#INFO} log message without PII.
+     *
+     * @param tag           Used to identify the source of a log message. It usually identifies the
+     *                      class or activity where the log call occurs.
+     * @param correlationID Unique identifier for a request or flow used to trace program execution.
+     * @param message       The message to log.
+     * @param exception      A related exception, or null.
+     */
+    public static void info(final String tag,
+                            @Nullable final String correlationID,
+                            @Nullable final String message,
+                            @Nullable final Throwable exception) {
+        getInstance().log(tag, LogLevel.INFO, correlationID, message, exception, false);
+    }
```

Comment:
If we are doing this, perhaps do it for other levels as well such as WARN, and VERBOSE? Seems strange that throwable is only accepted on INFO and ERROR and not on VERBOSE and WARN

Reply:
> If we are doing this, perhaps do it for other levels as well such as WARN, and VERBOSE? Seems strange that throwable is only accepted on INFO and ERROR and not on VERBOSE and WARN

Is there some reason for this distinction?  This is completely optional, honestly, and if there is, I can drop this.

Reply:
I think the question is "why isn't it even optional for VERBOSE and WARN"? - seems for these we strictly don't support passing a throwable. I personally think it should be supported on all levels as an optional parameter

Reply:
Lets make that separate.